src/__tests__/fire-event.test.tsx

```diff
@@ -443,4 +444,14 @@ describe('native events', () => {
     fireEvent(screen.getByTestId('test-id'), 'onMomentumScrollEnd');
     expect(onMomentumScrollEndSpy).toHaveBeenCalled();
   });
+
+  it('sets native state value for unmanaged text inputs', () => {
```

Comment:
[nit] it could be nice to put this test next to the other TextInput test to quickly see the difference